### PR TITLE
Created the Heptagram Bot Coming Soon Page

### PR DESCRIPTION
**Pull request by MidouWebDev**
I created the coming soon page. It is fully responsive !

### CODE REVIEW

**Comment by j-dogcoder** [comment]
Going to keep issue #1 open!

**Comment by j-dogcoder** [comment]
@MidouWebDev FYI, you have push/write access to the repo, so feel free to do work on a branch and then PR. (no need to fork!) This goes for all of the @Heptagram-Bot/website-developers team.

**Comment by MidouWebDev** [comment]
@j-dogcoder ok, got it !

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                </a>
+            </div>
+            <div class="discord-icon">
+                <a href="https://discord.gg/HSupF99kpq">
+                    <i class="fab fa-discord fa-2x"></i>
+                </a>
+            </div>
+        </div>
+        <div class="attribution">
+            <p>Designed and coded by <a href="https://github.com/MidouWebDev">MidouWebDev</a> (one of the Heptagram Developers)</p>
```
Comment (j-dogcoder) [inline index.html:40]: Hey @MidouWebDev, love giving you some credit, but can we also give some to the team? Maybe something like _**Designed and coded by the Heptagram Team**_?

Comment (j-dogcoder) [reply index.html:40]: Also totally fine with keeping you in as well, just want to give some credit to the rest of the team. (Especially once we get the full site in)

File: renovate.json [hunk 2]
```diff
@@ -1,5 +0,0 @@
-{
```
Comment (j-dogcoder) [inline renovate.json:1]: This file must stay in the root directory.

**Review by j-dogcoder — CHANGES_REQUESTED** [review]
Thanks @MidouWebDev! The file structure looks a lot cleaner, and I am loving the new page. I just have a few things that need changing(see my comments below).

File: index.html (hunk 1 above)
Comment (MidouWebDev) [reply index.html:40]: Okey, no problem ! I am planning to create a Heptagram Website Developers Page to give credits to everyone.

File: renovate.json (hunk 2 above)
Comment (MidouWebDev) [reply renovate.json:1]: Okey

**Comment by MidouWebDev** [comment]
@j-dogcoder i am going to commit changes right now

**Review by j-dogcoder — APPROVED** [review]

**Comment by j-dogcoder** [comment]
🎆

**Comment by j-dogcoder** [comment]
Good work!

**Comment by MidouWebDev** [comment]
@j-dogcoder thank you

**Pull request merged**